tools/toolchains/arm.py

```diff
@@ -568,6 +575,13 @@ def __init__(self, target, *args, **kwargs):
             if "--library_type=microlib" not in self.flags['asm']:
                 self.flags['asm'].append("--library_type=microlib")
 
+        stdio_lib = "std"
```

Comment:
```suggestion
        if hasattr(target, "stdio_lib") and target.stdio_lib == "small":
            self.flags["common"].append("-DMBED_MINIMAL_PRINTF")
```

`stdio_lib` and `"std"` are not needed. The special case is if `target.stdio_lib` is `"small"`. Is the check for the attribute existence necessary?